### DISCUSSION TITLE
For document need plugin of attached files manager

### DISCUSSION THREAD

**Discussion by sergeynilov** [filamentphp/filament#14100] [Help] [answered]
### Package

Other

### Package Version

^3.2-stable

### How can we help you?

In laravel 11 / filamentphp 3.2 app I make some kind of document with diffrent info and attached / view files(images, psd, docx).
I wonder are there some kind of of files manager plugin or convinient decision for it ?

**Comment by leandrocfe** [comment, marked as answer]
You can check the plugins section: https://filamentphp.com/plugins
